src/pyopensky/schema.py

```diff
@@ -410,16 +410,22 @@ class PositionData4(Base):
     maxtime: Mapped[pd.Timestamp] = mapped_column(UTCTimestampFloat)
     msgcount: Mapped[int]
     icao24: Mapped[str]
-    message: Mapped[str]
```

Comment:
Those columns are not available in position_data4 and I added the ones that are available but not yet included.

Also, in the trino.py, when using rawdata function, only the basic columns are used (mintime, maxtime, rawmsg, msgcount, icao24, sensors). In the prev pyopensky, when it was still for Impala, I downloaded the tables with those extra columns. I don't need all, I just need the lat, lon, and alt, so I don't have to re-calculate it from the rawmsg.

Reply:
ok, I guess they came from a wrong copy paste :(
I don't often use those tables, so I may have to trust you on these :sweat_smile:

Reply:
Totally understandable, in my opinion this part is a bit laborious but I can contribute 😄 

The trino.py part is a bit tricky because the columns varies between tables, I'll work on that one after updating the column names for each tables.

Here's a screenshot of the tables that I downloaded, I selected only what I needed.

<img width="734" alt="Screenshot 2023-11-10 at 14 35 35" src="https://github.com/open-aviation/pyopensky/assets/28396602/8c9ee849-7bcd-4cfc-92ac-c9e3fe775cc7">

Reply:
Thanks! I can provide a code review and guinea pig testing :)

Reply:
Will appreciate that!

Btw, you work at Onera right? I studied at Supaero and lived in the student housing next to Onera during my master. Sad that I can't make it in time to submit a paper for the conference in Toulouse. But I look forward to the next one :)

Reply:
We can set up a call if you need help understanding the code btw...

Reply:
That would be a good start for me!

I'm available next week on Monday to Thursday in the morning. Feel free to send me the invitation to m.f.rahman@tudelft.nl

Reply:
I sent you an email